mlibrary/catalog-browse
# February 2023 dependency updates.

# Overview
Monthly dependency updates to help maintain Catalog Browse.

## NPM
`npm`, `cssnano`, and `postcss` have been updated to their latest versions (as of February 7, 2023).

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Install the updated packages.
- Run the site and make a css change.
  - Build the CSS.
  - Make sure everything works.
